recipes/imagl/all/conanfile.py

```diff
@@ -14,8 +14,8 @@ class ImaglConan(ConanFile):
     description = "A lightweight library to load image for OpenGL application."
     topics = ("opengl", "texture", "image")
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False], "with_png": [True, False]}
-    default_options = {"shared": False, "fPIC": True, "with_png": True}
+    options = {"shared": [True, False], "fPIC": [True, False], "with_png": [True, False], "with_jpeg": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_png": True, "with_jpeg": True}
```

Comment:
please delete the option in the `configure` when it's not supported

Reply:
in `config_options()`

Reply:
```
def config_options(self):
    if tools.Version(self.version) < "0.2.0"
        del self.options.with_jpeg
```

Reply:
Done in my last commit